examples/local-access/setup-external-connection.ipynb

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
<p>Thoughts:</p><ul><li>Is it too much to instruct users to just use our Docker image for this?  </li><li>Should we be hard-coding versions in to the quickstart? What if images change and we forget to update the quickstart? Maybe we can use <code>conda list</code>  to show up-to-date versions</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/121/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Well, this is an open question- we can try to get people to use our docker image as the kernel for their local jupyter environment, but because it's their local context there may be a lot of unknown unknowns as it were. 

The versions are to try and give them concrete advice about what I know works as far as versioning, since we have libraries pinned to specific versions in our image/s. But you're right that it could cause tech debt for us. I would expect that when we change the pins of versions then I may want to re-test this whole thing and ensure that there are no unexpected effects from that with specific regard to local connections.